java-coprocessor/src/test/java/com/example/demo/DemoApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.example.demo;
```

Comment:
Do we need this file for the example?

Reply:
we sure don't! I'm a bit hesitant to get rid of it because it's boilerplate added by SpringBoot, and I don't want to remove any of the SpringBoot boilerplate for fear that it'll be more of a hassle to add it in manually or rerun the Spring Initializr